nixos/doc/manual/man-nixos-rebuild.xml

```diff
@@ -24,10 +24,18 @@
       <arg choice='plain'><option>build</option></arg>
       <arg choice='plain'><option>dry-build</option></arg>
       <arg choice='plain'><option>dry-activate</option></arg>
+      <arg choice='plain'><option>build-ova</option></arg>
       <arg choice='plain'><option>build-vm</option></arg>
       <arg choice='plain'><option>build-vm-with-bootloader</option></arg>
     </group>
     <sbr />
+    <arg>
+      <group choice='opt'>
+        <arg choice='plain'><option>--configuration</option></arg>
```

Comment:
I think it's nice because that is more to type than `nixos-rebuild -c /path/to/config build-vm`  or `nixos-rebuild build-vm -I nixos-config=/path/to/config`. And this doesn't make the code alot more complicated, it's just about 3 lines.